packages/shared.net.errors/src/PalantirApiError.ts

```diff
@@ -17,23 +17,23 @@
 export class PalantirApiError extends Error implements PalantirApiError {
   public message: string;
   public errorName?: string;
-  public errorType?: string;
+  public errorCode?: string;
```

Comment:
This is "errorCode" in the actual body:

https://github.com/palantir/osdk-ts/blob/76a85ef8138543a53f79af9a7d29bab54c9a3384/packages/shared.net.fetch/src/createFetchOrThrow.ts#L55-L63